bundle-size/app.js

```diff
@@ -130,28 +130,88 @@ function formatBundleSizeDelta(delta) {
   return 'Δ ' + (delta >= 0 ? '+' : '') + delta.toFixed(2) + 'KB';
 }
 
+/**
+ * Returns an explanation on why the check failed when the bundle size is
+ * increased.
+ *
+ * @param {number} baseRuntimeDelta bundle size delta.
```

Comment:
nit: units in JSDoc or variable name would be nice

Reply:
Done